lib/state/appActions.js

```diff
@@ -579,6 +588,11 @@ appActions.upload = function(uploadIndex, options, cb) {
   var self = this;
   cb = cb || _.noop;
 
+  var err = appActions._assertUploaderVersion();
```

Comment:
Might be un-necessary as hopefully they have been prompted to update before if there uploader is out of date